easybuild/easyconfigs/l/lagrangian-filtering/lagrangian-filtering-0.8.3-foss-2022a.eb

```
@@ -0,0 +1,83 @@
+# This installation unfortunately requires the GitHub dance, as the tarball
+# is simply not installing. For that reason there is no checksum! 
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'lagrangian-filtering'
+version = '0.8.3'
+
+homepage = 'https://github.com/angus-g/lagrangian-filtering'
+description = """Temporal filtering of data in a Lagrangian frame of reference.""" 
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('dask', '2022.10.2'),
+    ('h5py', '3.7.0'),
+    ('SciPy-bundle', '2022.05'),
+    ('netcdf4-python', '1.6.1'),
+    ('scikit-learn', '1.1.2'),
+    ('tqdm', '4.64.0'),
+    ('zarr', '2.13.3'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('parcels', '2.4.0', {
+        'checksums': ['221ce49b6cb5bb4996aa3c817c0de7f022c952a4d8cb4160ca7df7dd940714b5'],
+    }),
+    ('xarray', '2022.9.0', {
+        'checksums': ['a2a5b48ec0a3890b71ef48853fe9d5107d2f75452722f319cb8ed6ff8e72e883'],
+    }),
+    ('pykdtree', '1.3.6', {
+        'checksums': ['780b693d0555b857d7aab31e35d4293bf4ebdb9dec7a45ba4bb23b4400f626dc'],
+    }),
+    ('sosfilt', '1.2.0', {
+        'checksums': ['b3083fd2edf723a6163f00ddc36ec3ca20fbf6112cc68573dad9356544a9d734'],
+    }),
+    ('psutil', '5.9.4', {
+        'checksums': ['3d7f9739eb435d4b1338944abe23f49584bde5395f27487d2ee25ad9a8774a62'],
+    }),
+    ('progressbar2', '4.2.0', {
+        'checksums': ['1393922fcb64598944ad457569fbeb4b3ac189ef50b5adb9cef3284e87e394ce'],
+        'modulename': 'progressbar',
+    }),
+    ('cgen', '2020.1', {
+        'checksums': ['4ec99d0c832d9f95f5e51dd18a629ad50df0b5464ce557ef42c6e0cd9478bfcf'],
+    }),
+    ('pymbolic', '2022.1', {
+        'checksums': ['b52f451dd0b9803e03de3320af3b7ce5723070061c6d231c00216f6da4259012'],
+    }),
+    ('pytools', '2022.1.12', {
+        'checksums': ['4d62875e9a2ab2a24e393a9a8b799492f1a721bffa840af3807bfd42871dd1f4'],
+    }),
+    ('python-utils', '3.4.5', {
+        'checksums': ['7e329c427a6d23036cfcc4501638afb31b2ddc8896f25393562833874b8c6e0a'],
+    }),
+
+    (name, version, {
+        'sources': [{
+            'filename': '%(name)s-%(version)s-commit.tar.gz',
+            'git_config': {
+                'url': 'https://github.com/angus-g/',
+                'repo_name': name,
+                'commit': 'b942ff3',
```

The repo has tags, we can directly use the version of this easyconfig to get the sources and avoid defining any commit:
```suggestion
                'tag': version,
```